README.md

```diff
@@ -66,7 +66,41 @@ The application is built automatically and published on [Cozy Registry](https://
 
 ### Android environment
 
-To be able to start the mobile app on an emulator or a real device, you can follow [this guide](https://gist.github.com/drazik/11dfe2014a6b967821df93b9e10353f4).
+First you need to export your local host IP address:
+
+```bash
+$ export DEV_HOST=[YOUR_LOCAL_IP_ADDRESS]
```

Comment:
some emulators require to use `localhost` or `10.0.2.2`. Are we referring to local ip (`192.168.x.x`) for real devices AND for emulators?

Reply:
yup